.gitignore

```diff
@@ -6,4 +6,7 @@ coverage/
 docker-compose-data/
 node_modules/
 out/
-package-lock.json
\ No newline at end of file
+package-lock.json
+.github
+jest.config.js
+.DS_Store
```

Comment:
Please don't add the .DS_Store and .github entries. 
We should only add the stuff that this repository creates that we don't want to commit. 

The other stuff is dependent on the specific user and should go in their own global gitignore :). 
See: https://stackoverflow.com/questions/7335420/global-git-ignore